config/locales/translation.nb.yml

```diff
@@ -3347,14 +3383,14 @@ nb:
     users_password_reset:
     users_request_password_reset:
     welcome_goodbye:
-    welcome_index:
+    welcome_index: Bike Index - Sykkelregistrering som fungerer
   money:
     currencies:
-      en:
-      en-GB:
-      es:
-      nb:
-      nl:
+      en: USD
+      en-GB: GBP
+      es: EUR
+      nb: kr
```

Comment:
I seem to have missed this line, I just updated it in translation io from 'kr' to 'EUR'. sorry seth